### PR TITLE
Update changes

### PR DESCRIPTION
**Pull request by Qstick**
#### Database Migration
No

#### Description
Fix duplicate change items in UI modal
Add Changes to Notifications for Discord, Notifiarr, Webhook, Custom Scripts

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

### CODE REVIEW

**Comment by bakerboy448** [comment]
Should we do changelog for email as well?

File: src/NzbDrone.Core/Notifications/Discord/Discord.cs [hunk 1]
```diff
@@ -317,6 +317,16 @@ public override void OnApplicationUpdate(ApplicationUpdateMessage updateMessage)
                                           {
                                               Name = "New Version",
                                               Value = updateMessage.NewVersion.ToString()
+                                          },
+                                          new DiscordField()
+                                          {
+                                              Name = "New",
```
Comment (RobinDadswell) [inline src/NzbDrone.Core/Notifications/Discord/Discord.cs:323]: These could get to long for a discord message couldn't they? I cannot remember the character limit but a master release as an example could be a big release with lots of new and fixed

Comment (Qstick) [reply src/NzbDrone.Core/Notifications/Discord/Discord.cs:323]: Yea, Probably should do a take 5 or something

Comment (RobinDadswell) [reply src/NzbDrone.Core/Notifications/Discord/Discord.cs:323]: > Yea, Probably should do a take 5 or something

take 5 and a link to github for further info?

Comment (austinwbest) [reply src/NzbDrone.Core/Notifications/Discord/Discord.cs:323]: Personally i build the entire thing, truncate > 2000 (1000 for new & 1000 for fixed) so as much as possible will fit incase there are a lot of short messages so more fit. Just as a thought.

**Pull request closed without merging**